test/xml-test.js

```diff
@@ -13,11 +13,11 @@ var transferOnlyResponse = fs.readFileSync("./test/bxml-responses/transfer.xml",
 var nestingResponse = fs.readFileSync("./test/bxml-responses/nesting.xml", "utf8");
 var chainingResponse = fs.readFileSync("./test/bxml-responses/chaining.xml", "utf8");
 
-describe("XML Builder", function () {
+describe("Builder", function () {
 	var myApp;
```

Comment:
If this var is being reinitialized every test, I think it should be moved into a more narrow scope.